src/components/Nav/Nav.js

```diff
@@ -1,9 +1,158 @@
 import React, { Component } from 'react';
+import styled from 'styled-components';
 
 class Nav extends Component {
   render() {
-    return <div />;
+    return (
+      <Header>
+        <NavTopWrapper>
+          <NavTopUl>
+            {TOPLINK.map(item => {
+              return (
+                <NavtopItem key={item.name}>
+                  <TopLink> {item.name}</TopLink>
+                </NavtopItem>
+              );
+            })}
+          </NavTopUl>
+        </NavTopWrapper>
+        <HeaderMain>
+          <Logo>
+            <LogoLink>SHOE-KREAM</LogoLink>
+          </Logo>
+          <HeaderMainRightWrapper>
+            <HeaderMainRightUl>
+              {STYLE.map(item => {
+                return (
+                  <HeaderMainRightItem key={item.name}>
+                    <HeaderMainRightLink>{item.name}</HeaderMainRightLink>
+                  </HeaderMainRightItem>
+                );
+              })}
+            </HeaderMainRightUl>
+            <SearchBox>
+              <SearchBoxLink>
+                <i class="fas fa-search" />
+              </SearchBoxLink>
+            </SearchBox>
+          </HeaderMainRightWrapper>
+        </HeaderMain>
+      </Header>
+    );
   }
 }
 
 export default Nav;
+
+const Header = styled.div`
+  box-sizing: border-box;
+  min-height: 100px;
+  margin: 0px;
+  padding: 0px;
+  z-index: 1000;
+  background-color: #fff;
+  color: #222;
+`;
+
+const NavTopWrapper = styled.div`
+  display: flex;
+  justify-content: right;
+  align-items: center;
+  height: 32px;
+  padding: 8px 40px;
+  box-shadow: 0 1px 0 0 rgb(0 0 0 / 10%);
+`;
+const NavTopUl = styled.ul`
+  display: flex;
+  margin-left: auto;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0px;
+  margin-inline-end: 0px;
+  padding-inline-start: 40px;
+`;
+
+const NavtopItem = styled.li`
+  list-style: none;
+  margin-left: 24px;
+`;
+
+const TopLink = styled.a`
```

Comment:
a 태그 말고 링크 컴포넌트 사용하셔야 할거 같아요
```suggestion
const TopLink = styled(Link)`
```